.tekton/kubectl-dry-run.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: kubectl-dry-run
+spec:
+  description: >-
+    A task to kubectl dry-run the catalog structures
+
+  workspaces:
+    - name: source
+      mountPath: /workspace/source
+
+  steps:
+    - name: kubectl-apply-check
+      # TODO: find an UBI with kubectl
+      image: docker.io/alpine/k8s:1.18.2
+      workingdir: $(workspaces.source.path)
+      script: |
+        #!/usr/bin/env bash
+        set -ex
+        for i in task/*/*/;do kubectl apply -f ${i} --dry-run=client;done
```

Comment:
FYI about dry-run=client from this [doc](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands) :


```
dry-run 		none 	Must be "none", "server", or "client". If client strategy, only print the object that would be sent, without sending it. If server strategy, submit server-side request without persisting the resource. 
```